test/client-checkout-integration-test.js

```diff
@@ -229,6 +242,44 @@ suite('client-checkout-integration-test', () => {
     });
   });
 
+  test('it resolves with a checkout on Client.checkout#updateShippingAddress', () => {
+    const checkoutId = checkoutShippingAddressUpdateV2Fixture.data.checkoutShippingAddressUpdateV2.checkout.id;
+
+    fetchMock.postOnce(apiUrl, checkoutShippingAddressUpdateV2Fixture);
+
+    return client.checkout.updateShippingAddress(checkoutId, shippingAddress).then((checkout) => {
+      assert.equal(checkout.id, checkoutId);
+      assert.equal(checkout.shippingAddress.name, 'Bob Norman');
```

Comment:
Similar to the `checkout.id`, can we grab the expected values from the fixtures?